packages/core/metrics/core-metrics-collectors-server-internal/src/cgroup/cgroup.ts

```diff
@@ -42,9 +42,13 @@ export class OsCgroupMetricsCollector implements MetricsCollector<OsCgroupMetric
         return {};
       }
 
-      const args = { cpuAcctPath: this.cpuAcctPath!, cpuPath: this.cpuPath! };
       // "await" to handle any errors here.
-      return await (this.isCgroup2 ? gatherV2CgroupMetrics(args) : gatherV1CgroupMetrics(args));
+      return await (this.isCgroup2
+        ? gatherV2CgroupMetrics(this.cpuAcctPath!)
+        : gatherV1CgroupMetrics({
+            cpuAcctPath: this.cpuAcctPath!,
+            cpuPath: this.cpuPath!,
+          }));
```

Comment:
Refactored the signatures about v1/v2 detection a bit to get closer to reality (v2 -> a single group path)